htdocs/web_portal/views/scopes.php

```diff
@@ -1,6 +1,75 @@
 <div class="rightPageContainer">
-    <h1>Resource Scoping</h1>
-    <br />
+
+    <div style="float: left;">
+        <h1 style="float: left; margin-left: 0em;">
+            Scopes
+        </h1>
+        <span style="clear: both; float: left; padding-bottom: 0.4em;">
+            Click on the name of a scope to <?=$params['UserIsAdmin'] ? "edit it and " : "";?> view objects with that scope tag.
+        </span>
+    </div>
+    <!-- hide add when read only or user is not admin -->
+    <?php if(!$params['portalIsReadOnly'] && $params['UserIsAdmin']):?>
+        <div style="float: right;">
+            <center>
+                <a href="index.php?Page_Type=Admin_Add_Scope">
+                <img src="<?php echo \GocContextPath::getPath()?>img/add.png" height="25px" />
+                <br />
+                <span>Add Scope</span>
+                </a>
+            </center>
+        </div>
+    <?php endif; ?>
+
+    <?php $numberOfScopes = sizeof($params['Scopes'])?>
```

Comment:
I rather agre with the comments at https://www.php.net/manual/en/function.sizeof.php and prefer count() instead of sizeof();
```suggestion
    <?php $numberOfScopes = count($params['Scopes'])?>
```

Reply:
sizeof() is used in quite a few places throughout the code, so it might be worth fixing them all at some point. I'll change it everywhere it appears in scopes.php and scope.php for now though.